paths/electron/packaging-apps/outline.md

```diff
@@ -1,3 +1,20 @@
-## Learning goals
+# Electron 102: Packaging Electron Apps
 
-Take a locally running app, and package it up to be downloaded natively on other computers
+Main goal: Going from local to something someone can download
+
+## Packaging for different environments
+- Needs for Windows, OSX, and Linux
+- Packaging can be done for each OS, but the UI isn't automatically made to look native in the process
+
+## Tools to use
+- How to do this with Electron
+- Other libraries to use
+
+## Adding an Icon
+- Including the icon image for the program 
```

Comment:
👍 This is a great idea. Are you thinking something [like this](https://useiconic.com/open)?  We could also give recommendations on what type of image is allowed (open or public domain) and what size would be best.